lib/docker.ts

```diff
@@ -51,24 +51,13 @@ class Docker {
     return await dockerPull.pull(registry, repo, tag, opt);
   }
 
-  public async isDockerExperimentalEnabled(): Promise<boolean> {
-    const output = await subProcess.execute("docker", ["system", "info"]);
-    return output.stdout.includes("Experimental: true");
-  }
-
   public async pullCli(targetImage: string, options?: DockerOptions) {
-    let platformOption: string = "";
+    const opts: string[] = ["pull", targetImage];
```

Comment:
I'm not sure how smart Docker is with adding the option _after_ the image name (it might, I just don't know and want to make sure it works)